internals/secrethub/tree.go

```diff
@@ -50,26 +50,44 @@ func (cmd *TreeCommand) Register(r command.Registerer) {
 	clause := r.Command("tree", "List contents of a directory in a tree-like format.")
 	clause.Arg("dir-path", "The path to to show contents for").Required().PlaceHolder(optionalDirPathPlaceHolder).SetValue(&cmd.path)
 
+	clause.Flag("full-paths", "Print the full paths of the directories and secrets.").Short('f').BoolVar(&cmd.fullPaths)
+	clause.Flag("no-indentation", "Print the content without indentation.").Short('i').BoolVar(&cmd.noIndentation)
+	clause.Flag("no-report", "Skip the report at the bottom.").BoolVar(&cmd.noReport)
```

Comment:
The Unix tree command has a `--noreport` flag (without a dash). Shall we rename this one or accept both alternatives?

I believe there's no native flag alias support in Kingpin, but we could register two flags?

Reply:
I think that rather the latter, such that we are compliant both to the kebab-case and to Unix's legacy naming. I'll handle that.

Reply:
This should now be fixed.